frontend/src/flashcard.tsx

```diff
@@ -47,7 +47,7 @@ type TiledTextProps = {
   text: string;
   reorderable: boolean;
 } & Pick<PaperProps, "bg" | "c" | "h" | "w" | "withBorder" | "shadow"> &
-  Pick<TextProps, "size" | "fw" | "ff">;
```

Comment:
not strictly necessary but seems like the `size` prob became unused when some things moved to css